crates/cubecl-core/src/frontend/plane.rs

```diff
@@ -87,6 +87,66 @@ pub mod plane_sum {
     }
 }
 
+/// Perform an inclusive sum operation across all units in a plane.
+#[allow(unused_variables)]
+pub fn plane_inclusive_sum<E: CubePrimitive>(value: E) -> E {
+    unexpanded!()
+}
```

Comment:
What's the difference with plane_sum? I would add that in the docs just so it's clearer which one is needed.